DATATYPES.md

```diff
@@ -33,62 +33,6 @@ The result set metadata currently just displays the Type ID for each column as a
-|              2705 | Set[Boolean]                     | SetBoolean                              |
-|              2706 | Set[Int8]                        | SetInt8                                 |
-|              2707 | Set[Float8]                      | SetFloat8                               |
-|              2708 | Set[Char]                        | SetChar                                 |
-|              2709 | Set[Varchar]                     | SetVarchar                              |
-|              2717 | Set[Varbinary]                   | SetVarbinary                            |
-|              2722 | Set[Binary]                      | SetBinary                               |
-|              2716 | Set[Numeric]                     | SetNumeric                              |
-|              2721 | Set[Interval Year]               | SetIntervalYear                         |
-|              2721 | Set[Interval Year to Month]      | SetIntervalYearToMonth                  |
-|              2721 | Set[Interval Month]              | SetIntervalMonth                        |
-|              2714 | Set[Interval Day]                | SetIntervalDay                          |
-|              2714 | Set[Interval Day to Hour]        | SetIntervalDayToHour                    |
-|              2714 | Set[Interval Day to Minute]      | SetIntervalDayToMinute                  |
-|              2714 | Set[Interval Day to Second]      | SetIntervalDayToSecond                  |
-|              2714 | Set[Interval Hour]               | SetIntervalHour                         |
-|              2714 | Set[Interval Hour to Minute]     | SetIntervalHourToMinute                 |
-|              2714 | Set[Interval Hour to Second]     | SetIntervalHourToSecond                 |
-|              2714 | Set[Interval Minute]             | SetIntervalMinute                       |
-|              2714 | Set[Interval Minute to Second]   | SetIntervalMinuteToSecond               |
-|              2714 | Set[Interval Second]             | SetIntervalSecond                       |
-|              2710 | Set[Date]                        | SetDate                                 |
-|              2711 | Set[Time]                        | SetTime                                 |
-|              2715 | Set[TimeTz]                      | SetTimeTz                               |
-|              2712 | Set[Timestamp]                   | SetTimestamp                            |
-|              2713 | Set[TimestampTz]                 | SetTimestampTz                          |
-|              2720 | Set[Uuid]                        | SetUuid                                 |
-| 45035996273705276 | geometry                         | Geometry                                |
-| 45035996273705278 | geography                        | Geography                               |
 |               300 | Row                              | Row                                     |
```

Comment:
The values for Row, Array and Map are still needed because of the way the server works.   In a future server version, we plan to add a way to completely turn off complex types.  Once the driver enables this, these three values can be removed.  If the driver enables, then all types (300-302 and 1505-2720) will be enabled for return by the server.